app.json

```diff
@@ -25,10 +25,19 @@
       "adaptiveIcon": {
         "foregroundImage": "./assets/adaptive-icon.png",
         "backgroundColor": "#FFFFFF"
-      }
+      },
+      "package": "dev.expo.dsokal.test.example"
     },
     "web": {
       "favicon": "./assets/favicon.png"
+    },
+    "plugins": [
+      "@config-plugins/detox"
+    ],
+    "extra": {
+      "eas": {
+        "projectId": "08f1ceb3-64bf-4718-8f62-c2d372974092"
```

Comment:
I'd suggest removing the `projectId`, and instructing users to clone this repo and run `eas build:configure` and `eas update:configure` to set up the example for their own Expo space.